crates/yakui-app/src/lib.rs

```diff
@@ -207,13 +207,6 @@ impl Graphics {
                 self.resize(*size);
             }
 
-            Event::WindowEvent {
-                event: WindowEvent::ScaleFactorChanged { new_inner_size, .. },
-                ..
-            } => {
-                self.resize(**new_inner_size);
-            }
```

Comment:
Let's make sure we continue handling this event since it's important for correct display-switching support on Windows.

Reply:
winit no longer exposes a new size value in this event. I presumed that the actual resize event covers it. Is there additional work needed?